pandas/tests/io/formats/test_format.py

```diff
@@ -1610,7 +1606,7 @@ def test_to_string_small_float_values(self):
 
         result = df.to_string()
         # sadness per above
-        if "{x:.4g}".format(x=1.7e8) == "1.7e+008":
+        if f"{1.7e8:.4g}" == "1.7e+008":
```

Comment:
can `_three_digit_exp` function be used here instead?
```suggestion
        if _three_digit_exp():
```